api/Metadata.json

```diff
@@ -0,0 +1,163 @@
+{
+  "api": {
+    "Metadata": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Metadata",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "13",
+            "prefix": "webkit"
+          },
+          "chrome_android": {
+            "version_added": "0.16",
+            "prefix": "webkit"
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": false
+          },
+          "opera_android": {
+            "version_added": false
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": false,
+          "deprecated": false
+        }
+      },
+      "modificationTime": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Metadata/modificationTime",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": "13",
+              "prefix": "webkit"
```

Comment:
I don't think this is doubly prefixed (e.g., `webkitMetadata.webkitmodificationTime`). I think this prefix can be removed